ccd-definition/CaseField/CareSupervision/caseDocuments.json

```diff
@@ -126,9 +126,9 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "courtBundleList",
+    "ID": "courtBundleListV2",
```

Comment:
DO we need to be careful about removing here?

Reply:
As the old field has been moved to the DEPRECATED file I think it's fine to do now